ramls/circulation-logs.raml

```diff
@@ -0,0 +1,36 @@
+#%RAML 1.0
+
+title: Circulation Audit Logs API
+baseUri: https://github.com/folio-org/mod-audit
+version: v1
+
+documentation:
+  - title: mod-audit API
+    content: This documents the API calls that can be made to query circulation audit logs records
+
+types:
+  logRecordCollection: !include logRecordCollection.json
+  errors: !include raml-util/schemas/errors.schema
+traits:
+  validate: !include raml-util/traits/validation.raml
+  searchable: !include raml-util/traits/searchable.raml
+  language: !include raml-util/traits/language.raml
+
+resourceTypes:
+  collection-get: !include raml-util/rtypes/collection-get.raml
```

Comment:
collection-get resources type is used to return item with a given ID. Here we are going to request list of items by query. So collection.raml resource should be used here.

Reply:
This API should return collection of logRecords.